extendr-api/Cargo.toml

```diff
@@ -20,6 +20,7 @@ libc = { version = "0.2", optional = true }
 ndarray = { version = "0.15.3", optional = true }
 num-complex = { version = "0.4", optional = true }
 serde = { version = "1.0", features = ["derive"], optional = true }
+faer = { version = "0.18.0", optional = true }
```

Comment:
I'm worried that if we have a hard fix to a version we will make it incompatible with other libraries. I run into this issue a lot when I work with arrow

Reply:
I cannot say I have much experience with Rust libraries, but reading [Dependency Resolution](https://doc.rust-lang.org/cargo/reference/resolver.html#dependency-resolution) it would seem it's ok for this use case, as I don't think it's expected to depend directly on this, and a downstream project that uses `faer` directly would build the two different versions. `ndarray` also seems to be the same in the file.

Reply:
I don't think it's safe to do so, as by following [SemVer](https://semver.org/#spec-item-4) it seems it could break at any time.

Reply:
I suppose this should be fine for now. If we encounter any issues we can always revisit it. Nothing like a little bit of pain to encourage fixing a codebase.